.editorconfig

```diff
@@ -2,14 +2,10 @@ root = true
 
 [*]
 indent_style = space
```

Comment:
```
indent_style = tab
indent_size = 2
charset = utf-8

trim_trailing_whitespace = true
insert_final_newline = true
max_line_length=160
```
it's all in master and looking good ;-)

Reply:
Ok, I can see tabs working fine with indent_size = 2, I kind of like this concept because of Go :)

I have a doubt about `max_line_length=160`. Is there a chance we could make it 80? There are quite some reasons for it:
- some people don't see too well (me included) and have to use bigger fonts
- it works for people with smaller screens
- it allows you to have two files open side-by-side
- https://github.com/prettier/prettier/issues/2375#issuecomment-312429225

Reply:
Regarding sizes, I also use bigger fonts. How about a compromise of 120? Thing is I am used to comprehensive class naming like `QuantumSplitResolveConfiguration` and that really takes up space.

Reply:
Ok, let's try it out. 120 fits on my screen, barely, but fits :)